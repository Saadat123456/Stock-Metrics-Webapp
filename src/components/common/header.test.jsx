import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import Header from './header';

test('Renders Correctly', () => {
  const history = createMemoryHistory();
  const { container } = render(
    <Router history={history}>
      <Header />
    </Router>,
  );
  expect(container.firstChild).toMatchSnapshot();
});
