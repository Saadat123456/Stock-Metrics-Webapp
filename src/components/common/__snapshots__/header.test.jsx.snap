// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<header
  class="topBar"
>
  <ul
    style="width: 100%;"
  >
    <li
      class="heading"
    >
      Crypto Price
    </li>
  </ul>
</header>
`;
